manage.py

```diff
@@ -118,12 +121,39 @@ def show_drill_progress(args):
     print(f"{args.phone_number}:")
     print(f"\tid={user.user_id}")
     print(f"\tseq={user.seq}")
-    print(f"\taccount_info={user.account_info}")
+    print(f"\tprofile={user.profile}")
     print(f"\tlast_interacted_time={user.last_interacted_time.isoformat()}")
     print(f"\tfirst_unstarted_drill_slug={progress.first_unstarted_drill_slug}")
     print(f"\tfirst_incomplete_drill_slug={progress.first_incomplete_drill_slug}")
 
 
+def replay_message_stream(args):
+    kinesis = boto3.client("kinesis")
+    stream_name = f"message-log-{args.stage}"
+    shards = kinesis.list_shards(StreamName=stream_name)["Shards"]
```

Comment:
assuming this only returns live shards and not those that have been retired due to splitting or joining?

Reply:
Correct